### PR TITLE
Downloading multiple assets in parallel example

### LINKED ISSUE #719: Provide example with async, parallelized downloads, with asset activation & download
We recently added asset activation and download, let's create an example of using async to parallelize many downloads at once.

This is basically combining [orders_create_and_download_multiple_orders.py](https://github.com/planetlabs/planet-client-python/blob/main/examples/orders_create_and_download_multiple_orders.py) with the [downloading an asset example](https://github.com/planetlabs/planet-client-python/blob/main/docs/guide.md#downloading-an-asset) in the docs user guide to create a new script in the examples folder.

### PR DESCRIPTION
**Pull request by kevinlacaille**
Created an example of downloading multiple assets in parallel. This example is an expansion on the example given in [the guide](https://github.com/planetlabs/planet-client-python/blob/main/docs/guide.md#downloading-an-asset).

The script takes in two distinct PSScene item IDs with different asset types, gets their asset descriptions, activating them, waiting for them to become active, downloads them, then validating the checksums of downloaded files, in parallel.

Closes #719

### CODE REVIEW

**Comment by kevinlacaille** [comment]
A test for a randomly generated number fails, see #723.

File: examples/download_multiple_assets.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+"""Example of downloading multiple assets in parallel
+
+This is an example of getting two assets, activating them, waiting for them to
+become active, downloading them, then validating the checksums of downloaded
+files.
+
+[Planet Explorer](https://www.planet.com/explorer/) was used to define
```
Comment (jreiberkyle) [inline examples/download_multiple_assets.py:20]: Is this true for this script and where are AOIs used here?

**Review by jreiberkyle — APPROVED** [review]
Looks great runs great! My one ask is that the PR title be phrased for a Changelog entry

File: examples/download_multiple_assets.py (hunk 1 above)
Comment (kevinlacaille) [reply examples/download_multiple_assets.py:20]: Certainly was for the wildfire AOI, unsure about the river AOI, I took that from the guide, but I assume so.

**Pull request merged**